lib/Auth/Source/Discourse.php

```diff
@@ -102,7 +102,9 @@ public function authenticate(&$state)
         $discourseSsoUrl = $this->url . "/session/sso_provider?$query";
 
         // Redirect user to Discourse SSO
-        Utils\HTTP::redirectTrustedURL($discourseSsoUrl);
+	$httpUtils = new Utils\HTTP();
+	$httpUtils->redirectTrustedURL($discourseSsoUrl);
+	
```

Comment:
Can you use the same indenting as the other lines please?

```suggestion
        $httpUtils = new Utils\HTTP();
        $httpUtils->redirectTrustedURL($discourseSsoUrl);
```